src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+              processing the entire worlds sewage
+            </a>
+            , and 100% of the world are involved in that. The price is
+            unrealistically low due to taxpayers subsidizing the costs, and why
+            should we all pay for 15% of people to jet about on fancy holidays,
+            putting all of our health and existence at risk.
+          </p>
+
+          <p>
+            Offsetting cannot make flying sustainable, and airlines suggesting
+            so are just trying fix their reputation with the increasingly
+            climate-conscious public, and are doing so by passing the cost on to
+            you. They might pocket your money by doing literally nothing, or
+            maybe they'll do "not enough" with it, but offsetting definitely is
+            not the answer here. We need an aviation carbon tax or at the very
+            least a "frequent flier tax", which will raise flying to a realistic
+            price and in turn provide funding for electric trains.{' '}
+          </p>
+
+          <p>
+            Flying could be 25% of your carbon footprint for the year, so it's
+            probably the lowest hanging fruit for personal change. Climate
+            activists like George Manibot reduce their flying to once every
+            three years, but many Americans fly more than once a month, and no
+            amount of money can soak up that rate of emission.
+          </p>
+
+          <p>
+            Airlines aren't the only polluting industry excited to market
+            offsetting as a solution to their bad press, petrol stations are
+            getting in on the greenwashing too.
+          </p>
+
+          <img
+            src={imgShell}
+            alt="Shell poster promoting an app where you can carbon offset your driving"
+            style={{ width: '100%' }}
+          />
+
+          <p>
+            Just because the polluting industries support offsetting, doesn't
+            make offsetting a bad idea. They're generally pushing a weak
+            unprovable cheap version of it to benefit their marketing more than
+            anything else. To avoid getting caught out, they generally like to
+            push the least provable approaches to offsetting too.
+          </p>
+
+          <h2 id="morethanjusttrees">More Than Just Trees</h2>
+
+          <p>
+            Some organizations offering to offset CO2 will only talk about
+            trees, and this is a bit of a problem. For starters, reforestation
+            is the most difficult approach to regulate. Survival rate of the
+            trees from weather, beetles, disease, mold, ivy, deer, rabbits, etc
+            is hard to prove. Care can be taken to protect against many of these
+            impacts, but that costs money, so you need a reforestation that
+            actively cares about the outcome, instead of one which is trying to
```

Comment:
... but that **takes money and expertise**, so you need a reforestation **team** that actively...